packages/wonder-blocks-form/src/components/labeled-text-field.js

```diff
@@ -10,6 +14,8 @@ import TextField, {type TextFieldType} from "./text-field.js";
 type WithForwardRef = {|forwardedRef: React.Ref<"input">|};
 
 type Props = {|
+    ...AriaProps,
```

Comment:
This means that all aria props would be accepted, but we're only passing down `aria-describedby`.